slycot/tests/test.py

```diff
@@ -64,9 +58,5 @@ def test_td04ad_static(self):
             nr,a,b,c,d = transform.td04ad(rc,nin,nout,index,den,num)
 
 
-def suite():
-   return unittest.TestLoader().loadTestsFromTestCase(TestConvert)
```

Comment:
This is an old artifact, right? Nothing calls this function and it would fail because of undefined `TestConvert`.

Same exists in many other test scripts here.

Reply:
I see I introduced that in 9a098d12.  I don't remember why anymore, but I see it's been cargo-culted everywhere else.  From  file:///usr/share/doc/python3/html/library/unittest.html#organizing-test-code it looks like this `suite` function *could* introduce problems, so yes, let's erase them.  Thanks!